api/blocks/blocks.go

```diff
@@ -47,6 +54,16 @@ func (b *Blocks) handleGetBlock(w http.ResponseWriter, req *http.Request) error
 		return err
 	}
 
+	if raw == "true" {
+		rlpEncodedSummary, err := rlp.EncodeToBytes(summary)
+		if err != nil {
+			return err
+		}
+		return utils.WriteJSON(w, &JSONRawBlockSummary{
```

Comment:
Can we create a unit test to make the API call and decode the response body?

Reply:
was going to ask about tests - do we need also to update e2e tests for it ?

Reply:
Yeah, I think so and also internal unit tests in thor

Reply:
https://github.com/vechain/thor/pull/899/commits/dc802e2fed08c72cd5f6290ae5ff8b77f7165a43